oddt/metrics.py

```diff
@@ -190,3 +191,92 @@ def standard_deviation_error(y_true, y_pred):
     y_ = A * y_pred + B
     sd = (((y_true - y_) ** 2).sum() / (len(y_pred) - 1)) ** 0.5
     return sd
+
+
+def rie(y_true, y_score, alpha=20, pos_label=None):
+    """Computes Robust Initial Enhancement. This function assumes that results
+    are already sorted and samples with best predictions are first.
+
+    Parameters
+    ----------
+        y_true : array, shape=[n_samples]
+            True binary labels, in range {0,1} or {-1,1}. If positive label is
+            different than 1, it must be explicitly defined.
+
+        y_score : array, shape=[n_samples]
+            Scores for tested series of samples
+
+        alpha: float
+            Alpha. 1/Alpha should be proportional to the percentage in EF.
+
+        pos_label: int
+            Positive label of samples (if other than 1)
+
+    Returns
+    -------
+    rie : float
+         Robust Initial Enhancement
+
+    Notes
+    ----------
+    .. [1] Sheridan, R. P.; Singh, S. B.; Fluder, E. M.; Kearsley, S. K.
+           Protocols for bridging the peptide to nonpeptide gap in topological
+           similarity searches. J. Chem. Inf. Comput. Sci. 2001, 41, 1395−1406.
+
+    """
+    if pos_label is None:
+        pos_label = 1
+    labels = y_true == pos_label
+    alpha = float(alpha)
+    N = float(len(labels))
+    n = float(labels.sum())
+    r = np.argwhere(labels).astype(float) + 1  # need 1-based ranking
+    x = r / N
+    ra = float(n) / float(N)
+    return (np.exp(-alpha * x).sum()
```

Comment:
Can we keep the return statements short? We can create an out variable and return it.